backend/main.py

```
@@ -2,19 +2,24 @@
 from fastapi.responses import JSONResponse
 from loguru import logger
 
+from backend.email_generator.router import router as email_generator_router
 from backend.exceptions import APIException
+from backend.email_generator.router import router as email_generator_router
```

Ruff (0.11.9)

Redefinition of unused `email_generator_router` from line 5

Remove definition: `email_generator_router`

(F811)